SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorLayoutDetails.cpp

```diff
@@ -133,6 +134,131 @@ void FSpatialGDKEditorLayoutDetails::CustomizeDetails(IDetailLayoutBuilder& Deta
 				SNew(STextBlock).Text(FText::FromString("Push SpatialOS settings to iOS device"))
 			]
 		];
+
+	AssemblyNameInputErrorReporting = SNew(SPopupErrorText);
```

Comment:
Is it even useful to provide the ability to change these from editor settings? I suggest removing them from the settings window completely and only allow users to modify them via the Cloud Deployment Configuration window.

Reply:
that's a good point. I think I would prefer that as well.